pandas/tests/test_multilevel.py

```diff
@@ -2381,6 +2381,39 @@ def test_iloc_mi(self):
         tm.assert_frame_equal(result, expected)
 
 
+@pytest.fixture
+def sparse_df():
```

Comment:
move these to ``pandas/tests/sparse/test_reshape.py`` (new file); you can move other tests if appropriate (e.g. reshaping ones as well)